Changes

```diff
@@ -124,6 +125,12 @@ OCaml 5.0
   `caml_process_pending*` for multicore.
   (Guillaume Munch-Maccagnoni, review by Sadiq Jaffer and Gabriel Scherer)
 
+* #11337: pass 'flags' metadata to root scanners, to optimize stack
```

Comment:
The root-scanning hooks are internal, so this is not considered a breaking change.

Reply:
done, thanks!